apps/src/music/blockly/MusicBlocklyWorkspace.js

```diff
@@ -119,10 +121,10 @@ export default class MusicBlocklyWorkspace {
    *
```

Comment:
nit: can you update the doc as well to indicate that this only compiles and doesn't execute?

Reply:
Oops, I had meant to do that.  Fixed.